### PR TITLE
feat: oracle bing bong locally

### PR DESCRIPTION
**Pull request by davidterpay**
spins up local docker network composed of
1. blockchain node
2. prometheus dashboard
3. oracle side car

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,38 +30,26 @@ The slinky repository is composed of the following core packages:
...
+3. Host a prometheus instance that will scrape metrics from the oracle side-car. Navigate to http://localhost:9090 to see all network traffic and metrics pertaining to the oracle sidecar. Navigate to http://localhost:8001 to see all application-side oracle metrics.
 
-To see all network metrics, open a new terminal and run the following command and then navigate to http://localhost:9090:
+After a few minutes, run the following commands to see the prices written to the blockchain:
 
 ```bash
-$ make run-prom-client
-```
+# access the blockchain container
+$ docker exec -it slinky-blockchain-1 bash
```
Comment (aljo242) [inline README.md:49]: will it always be named this?

File: contrib/prometheus/prometheus.yml [hunk 2]
```diff
@@ -12,4 +12,4 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ["host.docker.internal:8000"]
+      - targets: ["oracle:8002"]
```
Comment (aljo242) [inline contrib/prometheus/prometheus.yml:15]: It it ok that we are pointing to 8001 and 8002 in some places?

Comment (davidterpay) [reply contrib/prometheus/prometheus.yml:15]: the 8001 is the app one, the 8002 one is the oracle side car metrics

File: README.md (hunk 1 above)
Comment (davidterpay) [reply README.md:49]: yea, docker compose gives it nice names

**Review by aljo242 — APPROVED** [review]

**Pull request merged**